gst/nnstreamer/tensor_typedef.h

```diff
@@ -131,7 +132,9 @@ typedef enum _nns_tensor_type
 typedef enum _nns_media_type
 {
   _NNS_VIDEO = 0, /**< supposedly video/x-raw */
+#ifndef NO_AUDIO
```

Comment:
We cannot allow this. The enum values MUST be consistent to all architectures & platforms.